extensions/amp-story/1.0/amp-story.js

```diff
@@ -1675,16 +1725,18 @@ export class AmpStory extends AMP.BaseElement {
     return this.getPageById(nextPageId);
   }
 
-
   /**
+   * Gives unsupported browser state
```

Comment:
This comment is slightly in conflict with the `@return` statement. Remove or reword to make it more consistent.